pkgs/development/libraries/libvncserver/default.nix

```diff
@@ -29,12 +30,16 @@ stdenv.mkDerivation rec {
     cmake
   ];
 
+  cmakeFlags = [
+    "-DWITH_SYSTEMD=${if withSystemd then "ON" else "OFF"}"
+  ];
+
```

Comment:
It should probably be moved under `propagatedBuildInputs`. Otherwise, LGTM.

Reply:
I've grepped through nixpkgs before writing this and found that most put this either directly before or after `nativeBuildInputs`, but I can of course move it around. It's just not clear to me what is the convention.

Reply:
I think it's common practice to put `flags` after `buildInput` blocks, and committers do check that, albeit undocumented. Other orders may be less obvious, regarding `preConfigure` and `flags`, for example.

Reply:
I would not be confident that there is a clear order, but I think there is a preference that «similar» declarations stay together.